### PR TITLE
Docs basically done, need review

### PR DESCRIPTION
**Pull request by jackbrighton**
Please give them a look-though to make sure everything is covered well.

### CODE REVIEW

File: docs/adding-ads.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Adding Super Cool Ads for Display in Posts
+
+After you [configure the plugin](configuration.md) to set the number of paragraphs before each ad insertion, and the number of available ad insertions, visit **Appearance** > **Widgets** to create the ads themselves. You'll see a number of Inserted Ad Positions as Widget areas where you can place widgets. You can set the number of available Inserted Ad Positions in the [plugin options panel](configuration.md). Each Inserted Ad Position widget area can have one and only one ad.
```
Comment (benlk) [inline docs/adding-ads.md]: > Each Inserted Ad Position widget area can have one and only one ad.

Technically false. They can have a bajillion ads or other widgets in them. But there *should* only be one per inserted position.

File: docs/configuration.md [hunk 2]
```diff
@@ -1,17 +1,22 @@
 # Configuration
 
-## The SCAIP settings page
+After installing the Super Cool Ad Inserted Plugin, a link to Ad Inserter configuration panel will appear in the WordPress Plugins menu:
```
Comment (benlk) [inline docs/configuration.md]: > Inserted

Inserter, not Inserted

> link to Ad Inserter

to the Ad

File: docs/adding-ads.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+To create an ad, begin by adding a Text Widget to one of the numbered Inserted Ad Positions:
```
Comment (benlk) [inline docs/adding-ads.md]: This area should note that for demonstration purposes, we're using a text widget, but other widget types are supported. Examples can be an ad widget from [DoubleClick for WordPress](https://wordpress.org/plugins/doubleclick-for-wp/), a newsletter signup widget, an image widget, a related posts widget, or anything else.

**Comment by aschweigert** [comment]
@benlk just make the changes directly to the docs instead of adding comments, happy to give it a final review and line edit when it's pretty much ready to go

File: docs/adding-ads.md (hunk 1 above)
Comment (jackbrighton) [inline docs/adding-ads.md]: Could change the last line to say "Each Inserted Ad Position should have only one ad."

File: docs/adding-ads.md (hunk 3 above)
Comment (jackbrighton) [inline docs/adding-ads.md]: Yep, let's add that to the docs.

File: docs/configuration.md (hunk 2 above)
Comment (jackbrighton) [inline docs/configuration.md]: good catch

**Pull request merged**